mizar/dp/mizar/operators/dividers/dividers_operator.py

```diff
@@ -72,6 +74,8 @@ def set_divider_provisioned(self, div):
         div.update_obj()
 
     def update_divider_with_bouncers(self, bouncer, net):
+        net.set_cluster_gateway(get_cluster_gateway_host_ip(self.core_api))
+
```

Comment:
please remove this blank line

Reply:
Removed

Reply:
it is still there.

Reply:
Not sure why it has not been pushed. I just push the change again and it is removed.